src/FMI2/ctype.jl

```diff
@@ -84,15 +84,16 @@ export fmi2VariableDependency
 # Custom helper, not part of the FMI-Spec. 
 fmi2Unknown = fmi2VariableDependency
 
-abstract type fmi2ModelDescriptionVariable end
+# custom abstract types, not part of the FMI-spec
+abstract type fmi2Attributes end # extended version of the Attributes used in fmi2XXXAttributes
+abstract type fmi2AttributesExt end # default version of the Attributes used in fmi2XXXAttributes
```

Comment:
What dos the `Ext` stand for? I would assume extended but that is what the comment behind `fmi2Attributes` indicates.